latest/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Redirecting</title>
+  <script>
+    var currentVersion = window.location.pathname.split("/")[1];
+    var xhr = new XMLHttpRequest();
+   
+    xhr.open("GET", "../versions.json");
+    xhr.onload = function () {
+      let versions = JSON.parse(this.responseText);
+
+      let res = versions.find(it => {
+        return it["aliases"].find(alias => { return alias == currentVersion; });
+      });
+
+      if(res.version){
+        window.location.pathname = "/" + res.version;
```

Comment:
Todo: replace the beginning of path with selected version using regex

Reply:
Something like:
```
window.location.pathname = window.location.pathname.replace(/^\/latest\//, `/${this.version}/`);
```
?